domain_access/src/DomainAccessManager.php

```diff
@@ -97,6 +95,7 @@ public static function getDefaultValue(FieldableEntityInterface $entity, FieldDe
     switch ($entity->getEntityType()) {
       case 'user':
       case 'node':
+        // @TODO $this is no accessible in a static context.
```

Comment:
Could be a linter hiccup, but from what I managed to find online, it has to do with the fact that `getDefaultValue()` is a static function and as per http://php.net/manual/en/language.oop5.static.php 

> Because static methods are callable without an instance of the object created, the pseudo-variable $this is not available inside the method declared as static.

I must admit I am not entirely sure what to do with it (if anything). Could use someone else's opinion.